docs/source/apis/Products.rst

```diff
@@ -0,0 +1,412 @@
+              marketplace_id=my_market,
+              skus=["OO-NL0F-795Z"],
+          )
+
+          for product in resp.parsed.Product:
+              product.CompetitivePricing.NumberOfOfferListings
+              product.CompetitivePricing.CompetitivePrices.CompetitivePrice.Price.LandedPrice.Amount
+
+   .. ########################################## GetCompetitivePricingForASIN ##########################################
+   .. automethod:: get_competitive_pricing_for_asin
+
+      .. rubric:: Example
+      .. code-block:: python
+
+          resp = products_api.get_competitive_pricing_for_asin(
+              marketplace_id=my_market,
+              asins=["B085G58KWT"],
+          )
+
+   .. ########################################## GetLowestOfferListingsForSKU ##########################################
+   .. automethod:: get_lowest_offer_listings_for_sku
+
+      .. rubric:: Example
+      .. code-block:: python
+
+          resp = products_api.get_lowest_offer_listings_for_sku(
+              marketplace_id=my_market,
+                skus=["OO-NL0F-795Z"],
+                condition="New" # Any, New, Used, Collectible, Refurbished, Club. Default = Any
+            )
+
+   .. ########################################## GetLowestOfferListingsForASIN #########################################
+   .. automethod:: get_lowest_offer_listings_for_asin
+
+      .. rubric:: Example
+      .. code-block:: python
+
+          resp = products_api.get_lowest_offer_listings_for_asin(
+              marketplace_id=my_market,
+              asins=["B085G58KWT"],
+              condition="New" # Any, New, Used, Collectible, Refurbished, Club. Default = Any
+          )
+
+   .. ########################################## GetLowestPricedOffersForSKU ###########################################
+   .. automethod:: get_lowest_priced_offers_for_sku
+
+      .. rubric:: Example
+      .. code-block:: python
+
+          resp = products_api.get_lowest_priced_offers_for_sku(
+              marketplace_id=my_market,
+              skus=["OO-NL0F-795Z"],
+              condition="New" # Any, New, Used, Collectible, Refurbished, Club. Default = Any
+          )
+
+   .. ########################################## GetLowestPricedOffersForASIN ##########################################
+   .. automethod:: get_lowest_priced_offers_for_asin
+
+      .. rubric:: Example
+      .. code-block:: python
+
+          resp = products_api.get_lowest_priced_offers_for_asin(
+              marketplace_id=my_market,
+              asins=["B085G58KWT"],
+              condition="New" # Any, New, Used, Collectible, Refurbished, Club. Default = Any
+          )
+
+   .. ############################################### GetMyFeesEstimate ################################################
+   .. automethod:: get_my_fees_estimate
+
+      Accepts one or more :py:class:`FeesEstimateRequest <mws.models.products.FeesEstimateRequest>` instances as
```

Comment:
Note the `:py:class:` directive here. `FeesEstimateRequest` is autodoc'd further down the page already; and these `:py:class:` and `:py:meth:` directives create automatic links to the reference for those autodocs.

Because the autodoc for `FeesEstimateRequest` is filled in with some examples on how to build it, I opted for a slimmer example in this method's docs, to focus on the method itself.